css/src/help-center-metabox.scss

```diff
@@ -1,18 +1,20 @@
-.yoast-help-center {
+.postbox {
 	.yoast-help-center {
-		&__button {
-			border: none;
-			box-shadow: none;
-			text-transform: none;
-			text-decoration: underline;
-			color: #0073aa;
-			margin: 0;
-			min-height: 0;
-			font: inherit;
-			padding: 8px 0;
+		.yoast-help-center {
```

Comment:
Please combine this line with the line below:
`.yoast-help-center__button {`

Reply:
When putting the old selector nesting back the hover doesn't work because of the `.postbox` wrapping.